src/signing.js

```diff
@@ -7,6 +7,8 @@
 
 var actualMethods = {};
 
+export var fastSigning;
```

Comment:
OK, after a few tests it turns out that `const` does not ensure immutability after exporting. Check the following code:
```js
var StellarBase = require("stellar-base");
StellarBase.Networks.PUBLIC = "test";
console.log(StellarBase.Networks.PUBLIC);
```
I think babel ensures `const` immutability only within a single file.

I can try to use `Object.defineProperty` but the other `const`s need to be changed too so I think this should be done in a separate PR.